vignettes/preprocessing-data.Rmd

```diff
@@ -1,78 +1,24 @@
 ---
-title: "Preprocessing data"
+title: "Pre-processing data"
 author: "NEST CoreDev"
-date: "2022-04-20"
+date: "2022-07-28"
 output: rmarkdown::html_vignette
 vignette: >
-  %\VignetteIndexEntry{Preprocessing data}
+  %\VignetteIndexEntry{Pre-processing data}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
-"Data" preprocessing refers to the code which precedes the `teal` app initialization in `app.R`, i.e.:
+## Usage in teal apps
```

Comment:
I guess this way is still supported and important @gogonzo ?

Reply:
@gogonzo ?

Reply:
I'm fine with the limited content of the vignette. Yes `get_code` is still supported, we never had any complaints about this stuff. People can use `get_code` or specify code in `cdisc_data` manualy. Below we have a link to pending preprocessing-data vignette so this is fully explained.